distage/distage-config/src/main/scala/com/github/pshirshov/izumi/distage/config/ConfigModule.scala

```diff
@@ -10,6 +10,7 @@ case class ConfigInjectorConfig(enableScalars: Boolean = false)
 class ConfigModule(config: AppConfig, configInjectorConfig: ConfigInjectorConfig = ConfigInjectorConfig()) extends ModuleDef {
   make[ConfigInjectorConfig].from(configInjectorConfig)
   make[AppConfig].from(config)
+  many[RuntimeConfigReaderCodecs].add[RuntimeConfigReaderCodecs.RuntimeConfigReaderDefaultImpl.type]
```

Comment:
Could you please add an instance instead of a singleton type? `many[RuntimeConfigReaderCodecs].from(RuntimeConfigReaderCodecs.RuntimeConfigReaderDefaultImpl)`